### ISSUE TITLE
What's Next: Master

### ISSUE THREAD

**Issue by theck13** [wordpress-mobile/WordPress-Android#7707]
## Goals
- Help new users find their way around the interface
- Help users understand why they should take certain actions
- Help users have success when they create new websites

## Tasks
- [x] Fancy Snackbar to turn on What's Next ([issue](https://github.com/wordpress-mobile/WordPress-Android/issues/7708), [pull request](https://github.com/wordpress-mobile/WordPress-Android/pull/7714))
  - [x] Show after site creation
- [x] Step-by-step tutorials
  - [x] New Snackbar style for tutorial steps ([issue](https://github.com/wordpress-mobile/WordPress-Android/issues/7708), [pull request](https://github.com/wordpress-mobile/WordPress-Android/pull/7714))
    - [x] colored text w/ inline icon
  - [x] Individual Quick Start task Snackbars:
    - [x] Show Snackbar for the first task after user activates QuickStart.
      - [x] If they tap "not now" hide snackbar untill user returns to MySite screen.
    - [x] Show this snackbar only once. After that, they will see "Continue Quick Start" Snackbar (once), that will take them to the Quick Start task list.
  - [x] The seven tutorials ([pull request](https://github.com/wordpress-mobile/WordPress-Android/pull/7983)):
    - [x] Continue site setup
    - [x] View Site
    - [x] Choose a theme
    - [x] Customize
    - [x] Share your site
    - [x] Publish a post
    - [x] View Stats
- [x] Checklist progress activity  ([issue](https://github.com/wordpress-mobile/WordPress-Android/issues/7710), [pull request](https://github.com/wordpress-mobile/WordPress-Android/pull/7844))
  - [x] Link to Checklist screen from My Site tab ([issue](https://github.com/wordpress-mobile/WordPress-Android/issues/7709), [pull request](https://github.com/wordpress-mobile/WordPress-Android/pull/7781))
  - [x] DB layer in Flux-C  ([issue [WordPress]](https://github.com/wordpress-mobile/WordPress-Android/issues/7784), [pull request [WordPress]](https://github.com/wordpress-mobile/WordPress-Android/pull/7822), [pull request [FluxC]](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/818))
- [x] Notification/Award ([issue](https://github.com/wordpress-mobile/WordPress-Android/issues/7711), pull request)

## Pre-release Tasks
- [x] Change default QuickStart enabled flag from true to false in AppPrefs.
- [x] Check if QuickStart completed before marking tasks as done passively.
- [x] Remove code that resets quick start at the start of the process and shows quick start dialog when changing site.
- [x] Reference the latest version of flux-c.

## Analytics
([issue](https://github.com/wordpress-mobile/WordPress-Android/issues/7712), pull request)
Analytics will be considered as part of every task.
- [ ] `whats_next_dialog_seen`
- [ ] `whats_next_dialog_negative_tapped`
- [ ] `whats_next_dialog_positive_tapped`
- [ ] `whats_next_list_accessed`
- [ ] `whats_next_list_create_site_tapped`
- [ ] `whats_next_list_view_site_tapped`
- [ ] `whats_next_list_browse_themes_tapped`
- [ ] `whats_next_list_customize_site_tapped`
- [ ] `whats_next_list_add_social_tapped`
- [ ] `whats_next_list_publish_post_tapped`
- [ ] `whats_next_list_follow_site_tapped`
- [ ] `whats_next_list_skip_all_tapped`
- [ ] `site_whats_next_complete`
- [ ] `whats_next_list_complete_seen`
- [ ] `notifications_whats_next_received`
- [ ] `notifications_whats_next_opened`
- [ ] `whats_next_request_seen`
- [ ] `whats_next_request_negative_tapped`
- [ ] `whats_next_request_positive_tapped`
- [ ] `whats_next_dialog_final_seen`
- [ ] `whats_next_dialog_final_negative_tapped`
- [ ] `whats_next_dialog_final_positive_tapped`

**Comment by khaykov** [comment]
Updated master issue description to align with iOS one (more checkboxes!)

**Comment by khaykov** [comment]
TODO before merging into `develop`:

- [ ] Change default QuickStart enabled flag from true to false in AppPrefs.
- [ ] Check if QuickStart completed before marking tasks as done passively.
